reactions.json

```diff
@@ -122,12 +122,18 @@
     "HCl": "chloroethane",
     "HBr": "bromoethane",
     "H2O / dil H2SO4": "ethanol",
-    "O3 / Zn dust / H2O": "methanal"
+    "O3 / Zn dust / H2O": "methanal",
+    "B2H6": "ethan-1-ol"

```

Comment:
ethan-1-ol is redundant, just ethanol is fine

Reply:
I did that because i saw `propan-1-ol` on line 71 & 72
Do you want me to change those as well?

Reply:
yea i took care of that

Reply:
Yeah i added another commit to remove 1-ol from it check latest commit @sleepbot-code

Reply:
I think its ok to be merged(?)